bitnami/solr/values.yaml

```diff
@@ -0,0 +1,655 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## String to partially override kafka.fullname template (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override solr.fullname template
+##
+# fullnameOverride:
+
+## Kubernetes Cluster Domain
+##
+clusterDomain: cluster.local
+
+## Add labels to all the deployed resources
+##
+commonLabels: {}
+
+## Add annotations to all the deployed resources
+##
+commonAnnotations: {}
+
+## Extra objects to deploy (value evaluated as a template)
+##
+extraDeploy: []
+
+## Number of solr instances to run
+##
+replicaCount: 3
+
+service:
+  type: ClusterIP
+  port: 8983
```

Comment:
If I remember correctly, Solr has a Web-based Admin UI. Then I guess it would make sense to have Ingress objects as well.

Reply:
That's true, I forgot about it. Good point!